contracts/README.md

```diff
@@ -63,6 +63,7 @@ The backbone of 𝚝𝟷, responsible for managing batches, state roots, and cro
 - **L2T1Messenger**: the L2 counterpart, handling incoming messages from L1 and sending outbound messages
```

Comment:
7683 folder is missing in the [Directory Structure](https://github.com/t1protocol/t1/blob/ba11660fae945a5bd62bad40335105bcd6ae3616/contracts/README.md#directory-structure)